python/cudf/cudf/utils/_numba.py

```diff
@@ -12,16 +12,14 @@
 # strings_udf. This is the easiest way to break an otherwise circular import
 # loop of _lib.*->cudautils->_numba->_lib.strings_udf
 @lru_cache
-def _get_cc_60_ptx_file():
+def _get_cuda_build_version_from_strings_udf():
```

Comment:
I think we should avoid associating the need for this information with strings specifically - the PTX file is needed for extended groupby apply support as well, and will soon contain numba runtime related functions. What about `_get_ptx_cuda_build_version` or `_get_udf_cuda_build_version` ?

Reply:
Renamed to `_get_cuda_build_version` for simplicity / alignment with the functions it calls.